### DISCUSSION TITLE
Is there a way to Override the plugin details in codeceptjs.config using the command line?

### DISCUSSION THREAD

**Discussion by jlabarrete** [PeterNgTr/codeceptjs-testrail#87] [Q&A] [answered]
Is there a way we can dynamically change or override  the plugin details in codeceptjs.config file through command line? 
In a way, I would like to change the "runName" or the "plan.name" ? 

By using a command line, 

npm run codeceptjs tests/ --runName "regression Test Sprint 1" 

I hope can override the value of "runName" in the plugin. 

Is this possible? 

I appreciate your reply to this.

**Comment by kobenguyent** [comment, marked as answer]
Hi @jlabarrete i could think of one fastest way to do so is using environment variable.
